test/ZNSRootRegistrar.test.ts

```diff
@@ -984,14 +984,48 @@ describe("ZNSRootRegistrar", () => {
       expect(finalstaked).to.equal(BigInt("0"));
       expect(finalToken).to.equal(ethers.ZeroAddress);
 
+      const protocolFee = getStakingOrProtocolFee(staked);
+
       // Verify final balances
-      const computedFinalBalance = balance + staked;
+      const computedFinalBalance = balance + staked - protocolFee;
       const finalBalance = await zns.meowToken.balanceOf(user.address);
       expect(computedFinalBalance).to.equal(finalBalance);
     });
   });
 
   describe("Revoking Domains", () => {
+    it("Sends a protocol fee as part of revoke flow", async () => {
+      await defaultRootRegistration({
+        user,
+        zns,
+        domainName: defaultDomain,
+        distrConfig: {
+          pricerContract: await zns.curvePricer.getAddress(),
+          paymentType: PaymentType.STAKE,
+          accessType: AccessType.OPEN,
+        },
+      });
+
+      const domainHash = await getDomainHashFromEvent({
+        zns,
+        user,
+      });
+
+      const [ price, stakeFee ] = await zns.curvePricer.getPriceAndFee(ethers.ZeroHash, defaultDomain, false);

```

Comment:
`protocolFee`, not `stakeFee`. we shouldn't confuse ourselves since these are different things.

Reply:
The value is correct still, it was just an effort to use one `await` call instead of two, to get the same value under a different name.